From 1151fa5edcf265b26fa66cee213508cd7934b262 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 1 Apr 2024 10:23:06 -0700
Subject: [PATCH] Push out update checks to 24 hours

Allow for more time before autoupdate triggering to reduce quality
escapes getting immediately picked up by everybody.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 data/swupd-update.timer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/swupd-update.timer b/data/swupd-update.timer
index 7144701..3f71efd 100644
--- a/data/swupd-update.timer
+++ b/data/swupd-update.timer
@@ -4,7 +4,7 @@ Documentation=man:swupd_update
 
 [Timer]
 OnBootSec=45min
-OnUnitActiveSec=45min
+OnUnitActiveSec=24h
 AccuracySec=300
 RandomizedDelaySec=2700
 
-- 
2.44.0

